helms/knative/templates/serving/activator/cm.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+        {{- include "knative.labels" (dict "root" . "name" "activator") | nindent 4 }}
+  name: config-tracing
+  namespace: {{ .Release.Namespace }}
+        {{- if .Values.config.tracing }}
```

Comment:
Please do not add spaces before control constructions in templates, it is harder to read

Reply:
Fixed